controllers/agentaction_controller.go

```diff
@@ -439,7 +444,7 @@ func (r *AgentActionReconciler) createAgentJob(ctx context.Context, log logr.Log
 						},
 					},
 					Volumes:            volumes,
-					RestartPolicy:      "Never", // TODO: Make the retry policy configurable on the Installation
+					RestartPolicy:      "Never",
```

Comment:
Can you include a comment with a link to that great k8's comment you linked in the associated issue for this PR that explains how RestartPolicy=Never works. That was some really good context that will help out whoever works on this code next.